etl/xml_importer/xpaths.py

```diff
@@ -110,15 +112,15 @@
     #MEASUREMENT
     "Artwork_Measurements_Path": 'lido:descriptiveMetadata/lido:objectIdentificationWrap/lido:objectMeasurementsWrap/lido:objectMeasurementsSet',
     "Measurement_DisplayName_Path": 'lido:displayObjectMeasurements',
-
+    "Measurement_Type_Path": 'lido:objectMeasurements/lido:measurementsSet/lido:measurementType',
+    "Measurement_Unit_Path": 'lido:objectMeasurements/lido:measurementsSet/lido:measurementUnit',
+    "Measurement_Value_Path": 'lido:objectMeasurements/lido:measurementsSet/lido:measurementValue',
+    "Measurement_Extend_Path": 'lido:objectMeasurements/lido:extentMeasurements',
+    "Measurement_Shape_Path": 'lido:objectMeasurements/lido:shapeMeasurements>',
```

Comment:
Remove the `>` at the end
```suggestion
    "Measurement_Shape_Path": 'lido:objectMeasurements/lido:shapeMeasurements',
```

Reply:
> is removed

Reply:
> is removed

Reply:
**>** is removed